spec.md

```diff
@@ -182,7 +182,8 @@ as a string.
 - `Binary` - Sequence of bytes.
   - String encoding: Base64 encoding per
     [RFC4648](https://tools.ietf.org/html/rfc4648).
-- `Map` - `String`-indexed dictionary of `Any`-typed values.
+- `Map` - `String`-indexed dictionary of `Any`-typed values. Only valid in the
```

Comment:
It's sufficient to say this only in one place normatively.

Reply:
I'm concerned about confusion between line 150 and later, but I'll remove if this seems clear to everyone.

> The following abstract data types are available for use in attributes.